package.json

```diff
@@ -8,27 +8,27 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "auto-changelog": "^1.11.0",
-    "autoprefixer": "^9.4.8",
-    "browserify": "^16.0.0",
-    "budo": "^11.3.2",
+    "auto-changelog": "^1.16.1",
+    "autoprefixer": "^9.6.1",
+    "browserify": "^16.5.0",
+    "budo": "^11.6.3",
     "cpx": "^1.5.0",
     "from2-string": "^1.1.0",
     "gh-release": "^3.5.0",
-    "js-yaml": "^3.12.0",
+    "js-yaml": "^3.13.1",
     "mkdirp": "^0.5.1",
-    "npm-run-all": "^4.1.3",
-    "parallel-transform": "^1.1.0",
-    "postcss": "^7.0.0",
-    "postcss-browser-reporter": "^0.5.0",
-    "postcss-cli": "^6.0.0",
-    "postcss-import": "^12.0.0",
-    "postcss-reporter": "^6.0.0",
+    "npm-run-all": "^4.1.5",
+    "parallel-transform": "^1.2.0",
+    "postcss": "^7.0.18",
+    "postcss-browser-reporter": "^0.6.0",
+    "postcss-cli": "^6.1.3",
+    "postcss-import": "^12.0.1",
+    "postcss-reporter": "^6.0.1",
     "postcss-url": "^8.0.0",
     "pump": "^3.0.0",
-    "rimraf": "^2.6.2",
-    "split2": "^3.1.0",
-    "swagger-ui": "^3.21.0"
+    "rimraf": "^3.0.0",
```

Comment:
is `rimraf` 3x same API?

Reply:
Yes it is. I also checked the other dependencies (in case they did not respect semver).